default/python/charting/README.md

```diff
@@ -0,0 +1,12 @@
+# Chars for AI
+Tool that parses the logs and build charts about AI
+
+## Pre-requirements
+```shell
+pip install requirements_charts.txt
+```
+
+## Run
+```shell
+streamlit run charting\streamlite_app.py
```

Comment:
My pip wanted `pip install -r requirements_charts.txt`
Also first command must be called from this directory, send from one further up and the path only works on windows. I suggest to remove `charting\`